### PR TITLE
Save/load images while developing

### PR DESCRIPTION
**Pull request by cmc333333**
This builds on #14.

While under development, use django to store and host images/media. Production will likely use `django-storages` or similar. Also adds some style tweaks so the images don't take over the page.

Media's stored and hosted from the "media" subdirectory of your working dir when you `runserver`.

### CODE REVIEW

File: peacecorps/peacecorps/models.py [hunk 1]
```diff
@@ -115,8 +115,7 @@ def __str__(self):
         return '%s' % (self.title)
 
     def url(self):
```
Comment (khandelwal) [inline peacecorps/peacecorps/models.py:118]: Should we make this a property of the model?

**Comment by cmc333333** [comment]
Made `url` a property - this makes the templates a little nicer, too.

@annalee let me know if this is holding you back on anything; I can separate out the file component from #14 and we can merge them separately.

**Comment by khandelwal** [comment]
@cmc333333 We can easily get Theresa to audit this later. I'm happy to merge if it's holding anyone back.

**Comment by annalee** [comment]
+1 on merge.

**Comment by khandelwal** [comment]
Conflicts up in here.

**Comment by cmc333333** [comment]
Merged upstream.

**Pull request merged**